tests/test_full_smrf.py

```diff
@@ -64,19 +138,93 @@ def setUpClass(cls):
         if isdir(cls.output):
             shutil.rmtree(cls.output)
 
-        run_smrf(cls.config_file)
+        config = copy(cls.base_config)
+        config.raw_cfg['system']['threading'] = False
 
-    def test_variables(self):
-        """
-        Compare that the entire output datasets to confirm they are the same as
-        the gold file provided.
-        """
-        variables = ['air_temp', 'precip_temp', 'net_solar',
-                     'percent_snow', 'precip', 'thermal', 'wind_speed',
-                     'wind_direction', 'snow_density', 'vapor_pressure']
+        config.apply_recipes()
+        config = cast_all_variables(config, config.mcfg)
+
+        run_smrf(config)
+
+    def test_air_temp(self):
+        """Test RME air_temp"""
```

Comment:
This is a personal view:
Test class name and the method already describe what is tested and I don't see a benefit having it again as the description with no additional information.

Reply:
It makes a nice output when running with with `-v` but the test name is pretty descriptive.